Source/OxyPlot/Svg/SvgRenderContext.cs

```diff
@@ -128,6 +122,20 @@ public override void DrawRectangle(OxyRect rect, OxyColor fill, OxyColor stroke,
             this.w.WriteRectangle(rect.Left, rect.Top, rect.Width, rect.Height, this.w.CreateStyle(fill, stroke, thickness), edgeRenderingMode);
         }
 
+        /// <summary>
+        /// Determines the font descent for the given font.
+        /// </summary>
+        /// <param name="fontFamily">The font family.</param>
+        /// <param name="fontSize">Size of the font.</param>
+        /// <param name="fontWeight">The font weight.</param>
+        /// <returns>A positive value, that distance between the baseline and the bottom of any rendered text.</returns>
+        public virtual double GetFontDescent(string fontFamily, double fontSize, double fontWeight)
+        {
+            var lineHeight = this.MeasureText("Ag", fontFamily, fontSize, fontWeight).Height;
```

Comment:
See comment in `FontMetrics.cs` - I think the name `lineHeight` might be misleading.

Reply:
Yep, though I think that method should be gone altogether.